File: JAVA/impar.java
import java.util.Scanner;

public class impar {
    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Qual número quer verificar?");
        int num1 = scanner.nextInt();

        String resultado = (num1 % 2 == 0) ? "O número é par" : "O número é impar";
        System.out.println(resultado);
    }
}

File: C/impar.c
// Calculadora que realiza as 4 operações básicas
// Soma, subtração, multiplicação e divisão


#include <stdio.h>

// Solicita a operação
int main(){
    int numero;
    printf("Qual o numero voce quer ver?\n");
    scanf("%d", &numero);

    printf("O numero e %s\n", (numero % 2 == 0  ? "par":"impar"));

    float num1;
    printf("Digite qualquer coisa para encerrar o programa. \n");
    scanf("%s", &num1);
    return 0;
}